<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Neuro MINE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear all fields</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Model Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>Predictor .csv Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="text">
     <string>Response .csv Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="text">
     <string>Use Time as a Predictor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="text">
     <string>Run Shuffle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="text">
     <string>Test Score Threshold:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="text">
     <string>Taylor Expansion Significance Threshold:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="text">
     <string>Taylor Expansion Look Ahead:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="text">
     <string>Fit Variance Fraction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="text">
     <string>Linear Fit Variance Fraction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="text">
     <string>Square Fit Variance Fraction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="text">
     <string>Model History:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="text">
     <string>Store Linear Receptive Fields (Jacobians):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="text">
     <string>Upload JSON File with Existing Parameters:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>231</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_9"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_10"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
